content/md/franklin-insights/franklin-insights-mappings.md

```diff
@@ -59,14 +59,20 @@ _Since the 3.1 version_, unmapped attributes are displayed first.
 
 ### Exact match
 
-No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, the proposal is automatically generated!
+No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
 
-![image](../img/franklin_exact_match.png)
+![Franklin's exact match screen](../img/franklin_exact_match.png)
 
 ::: info
-We consider attributes match if Franklin attribute code equals the PIM attribute code or PIM attribute label for an English catalog locale.
+We consider attributes match if the Franklin attribute code equals the PIM attribute code or PIM attribute label for an English catalog locale.
 :::
 
+### Suggested mapping
+
+No need to search through a long list of your catalog attributes: if they're close, we suggest the best match for the attribute. The best match is selected, and you can check the other close ones Franklin found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```

Comment:
Same here 'are' instead of its contraction.
Also suggest to use 'similar' and 'almost identical' instead of 'close'.
Didn't get the sentence "The best match is selected, and you can check the other similar ones Franklin found".

```suggestion
No need to search through a long list of your catalog attributes: if they are almost identical, we suggest the best match for the attribute. The best match is selected, and you can check the other similar ones Franklin found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```